src/main.rs

```diff
@@ -0,0 +1,112 @@
+use std::fs::File;
+use std::io::Write;
+use std::sync::Arc;
+
+use clap::{App, Arg, ArgMatches, SubCommand};
+use http::{Request, Response};
+
+use third_wheel::*;
+
+#[tokio::main]
+async fn main() -> SafeResult {
+    let matches = App::new("third-wheel")
```

Comment:
I hate `Clap` so much :smile: Check https://docs.rs/argh/0.1.3/argh/ for muuuch simpler CLIs. I think the main idea of the project was "optimising for less code" or something like that.